packages/components/inputs/money-input/src/money-input.tsx

```diff
@@ -805,7 +809,11 @@ const MoneyInput = (props: TMoneyInputProps) => {
             inputId={MoneyInput.getCurrencyDropdownId(moneyInputId)}
             name={getCurrencyDropdownName(props.name)}
             value={option}
-            isDisabled={props.isDisabled}
+            isDisabled={
+              props.isCurrencyInputDisabled
+                ? props.isCurrencyInputDisabled
+                : props.isDisabled
+            }
```

Comment:
Just so that I have something to comment, here is a little nitpicking 😄 :

```suggestion
isDisabled={props.isCurrencyInputDisabled || props.isDisabled}
```

Reply:
Yes, we can do this too. 
Updated here: https://github.com/commercetools/ui-kit/pull/2847/commits/588183887d960bb9caa00de97655d0e6ae6bee68
Thank you